Microsoft.Toolkit.Uwp.UI/ImageCache/ImageCache.cs

```diff
@@ -155,11 +138,12 @@ public static string GetCacheFileName(Uri uri)
         }
 
         /// <summary>
-        /// Load a specific image from the cache. If the image is not in the cache, ImageCache will try to download and store it. If image cannot be loaded an exception will be thrown.
+        /// Load a specific image from the cache. If the image is not in the cache, ImageCache will try to download and store it.
         /// </summary>
         /// <param name="uri">Uri of the image.</param>
+        /// <param name="throwOnError">Indicates whether or not exception should be thrown if imagge cannot be loaded</param>
         /// <returns>a BitmapImage</returns>
-        public static async Task<BitmapImage> GetFromCacheOrDieAsync(Uri uri)
+        public static async Task<BitmapImage> GetFromCacheAsync(Uri uri, bool throwOnError = false)
```

Comment:
Not sure if throwOnError is the right name for this type of parameter. Is there a "standard" one?